2-mandatory/2-financial-times.js

```diff
@@ -13,25 +20,72 @@ function potentialHeadlines(allArticleTitles) {
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+//        ["The", "three", "questions", "that", "dominate", "investment"]
+//         fewestNumberOfWords = articleSplitter.length;
+
 function titleWithFewestWords(allArticleTitles) {
+    let shortestHeadline;
```

Comment:
you could declare 'shortestHeadline' with const keyword instead let

Reply:
I think `shortestHeadline` needs to be declared with `let` here as we are re-assigning a new value to it a few lines later.